db.php

```diff
@@ -11,52 +11,53 @@ function getItems(object $db): array {
 function arraySplit(array $items) : string{
     $result = '';
     if (!count($items)) {
-        $result .= '<div>' . 'there aren\'t any database items to present' . '<div>';
+        $result .= '<div class = "empty">' . 'there aren\'t any database items to present' . '</div>';
```

Comment:
You don't actually need to nest your foreach loop inside of an else statement here. You can just have your first if statement and then start the foreach loop on the next line

Reply:
I have done this